packages/common/src/content/hostedServiceUtils.ts

```diff
@@ -86,10 +86,26 @@ export function isAGOFeatureServiceUrl(url: string): boolean {
   // TODO: we should really centralize this regex somewhere
   const FEATURE_SERVICE_URL_REGEX = /(feature)server(\/|\/(\d+))?$/i;
   return (
-    !!url && url.includes("arcgis.com") && FEATURE_SERVICE_URL_REGEX.test(url)
+    !!url &&
+    url.includes("arcgis.com") &&
+    FEATURE_SERVICE_URL_REGEX.test(url) &&
+    !isSecureProxyServiceUrl(url)
   );
 }
 
+/**
+ * Portal secure proxy services are identified by looking for these patterns in the `url`:
+ * - /sharing/servers/
+ * - /sharing/appservices/
+ * - /usrsvcs/servers/
+ * - /usrsvcs/appservices/
+ *
+ * @param url
+ */
+export function isSecureProxyServiceUrl(url: string): boolean {
```

Comment:
Let's either add tests or add a comment saying that we won't add tests since this was taken from the JSAPI